test/views/SelectTransactionsViewSpec.scala

```diff
@@ -42,6 +42,16 @@ class SelectTransactionsViewSpec extends ViewTestHelper {
         view.getElementById("start").text() mustBe dateText
       }
 
+      "start hint must be Displayed" in new Setup {
```

Comment:
Please update the test description to **"start hint is displayed"** as right now its reading like below when test is run. Same for line no 50

SelectTransactionsView
should display correct information
 - when start hint must be Displayed
 - when end hint must be dispalyed